manager/integration/tests/common.py

```diff
@@ -4133,14 +4145,14 @@ def expand_and_wait_for_pvc(api, pvc, size):
     api.patch_namespaced_persistent_volume_claim(
         pvc_name, 'default', pvc)
     complete = False
-    for i in range(RETRY_COUNTS):
+    for i in range(RETRY_COUNTS_LONG):
```

Comment:
Why need to increase time here? Is it due to some latency happening in reconciling?

Reply:
Due to this reason https://github.com/longhorn/longhorn-tests/pull/1344#issuecomment-1546815050